nncf/quantization/algo.py

```diff
@@ -1342,6 +1345,7 @@ def __init__(self, target_model: NNCFNetwork,
                  weight_quantizers: Dict[WeightQuantizerId, WeightQuantizerInfo],
                  non_weight_quantizers: Dict[NonWeightQuantizerId, NonWeightQuantizerInfo],
                  groups_of_adjacent_quantizers: GroupsOfAdjacentQuantizers,
+                 quantizer_setup: SingleConfigQuantizerSetup,
```

Comment:
Shouldn't extend the regular controller with a setup parameter right now. I understand that this might be required in the scope of quantizer setup export, but this should be done in a separate PR.

Reply:
ok, reverted